bullet_train-super_scaffolding/config/locales/en/scaffolding/completely_concrete/tangible_things.en.yml

```diff
@@ -118,107 +75,20 @@ en:
           one: One
           two: Two
           three: Three
       multiple_super_select_values:
-        _: &multiple_super_select_values Multiple Super Select Values
-        label: *multiple_super_select_values
-        heading: *multiple_super_select_values
+        heading: Multiple Super Select Values
         options:
           four: Four
           five: Five
           six: Six
       text_area_value:
-        _: &text_area_value Text Area Value
-        label: *text_area_value
-        heading: *text_area_value
+        heading: Text Area Value
       action_text_value:
-        _: &action_text_value Trix Editor Value
-        label: *action_text_value
-        heading: *action_text_value
+        heading: Trix Editor Value
       # 🚅 stop any skipping we're doing now.
       # 🚅 super scaffolding will insert new fields above this line.
-      created_at:
-        _: &created_at Added
-        label: *created_at
-        heading: *created_at
-      updated_at:
-        _: &updated_at Updated
-        label: *updated_at
-        heading: *updated_at
-    api:
-      collection_actions: "Collection Actions for Tangible Things"
-      index: "List Tangible Things"
-      create: "Add a New Tangible Thing"
-      member_actions: "Actions for an Individual Tangible Thing"
-      show: "Retrieve a Tangible Thing"
-      update: "Update a Tangible Thing"
-      destroy: "Delete a Tangible Thing"
-      fields: *fields
-    index:
-      section: "%{creative_concepts_possessive} Tangible Things"
-      contexts:
-        scaffolding/absolutely_abstract/creative_concept:
-          header: Tangible Things
-          description: Below is a list of Tangible Things that have been added for %{creative_concept_name}.
-          description_empty: No Tangible Things have been added for %{creative_concept_name}.
-      fields: *fields
-      buttons: *buttons
-    show:
-      section: "%{tangible_thing_name}"
-      header: Tangible Thing Details
-      description: Below are the details we have for %{tangible_thing_name}.
-      manage_description: You'll also find options for updating these details or removing %{tangible_thing_name} from %{creative_concept_name} entirely.
-      fields: *fields
-      buttons: *buttons
-    form: &form
-      buttons: *buttons
-      fields: *fields
-    new:
-      section: "New Tangible Thing for %{creative_concept_name}"
-      header: New Tangible Thing Details
-      description: Please provide the details of the new Tangible Thing you'd like to add to %{creative_concept_name}.
-      form: *form
-    edit:
-      section: "%{tangible_thing_name}"
-      header: Edit Tangible Thing Details
-      description: You can update the details or settings for %{tangible_thing_name} below.
-      form: *form
-    notifications:
-      created: Tangible Thing was successfully created.
-      updated: Tangible Thing was successfully updated.
-      destroyed: Tangible Thing was successfully destroyed.
-  account:
```

Comment:
We can generate this whole section for `account`, `admin`, and every other namespace at runtime by just copying over `scaffolding/completely_concrete/tangible_things`.